rust/schema/annotations.py

```diff
@@ -228,41 +228,36 @@ class _:
     loop_body: drop
 
 
-class CallExprBase(Expr):
-    """
-    A function or method call expression. See `CallExpr` and `MethodCallExpr` for further details.
-    """
-    arg_list: optional["ArgList"] | child
-    attrs: list["Attr"] | child
-    args: list["Expr"] | synth
-
-
-@annotate(CallExpr, replace_bases={Expr: CallExprBase}, cfg=True)
+@annotate(CallExpr, cfg=True)
 class _:
     """
-    A function call expression. For example:
+    NOTE: Consider using `Call` instead, as that includes all kinds of calls to
+    functions, and excludes instantiations of tuple structs and tuple enum variants.
```

Comment:
"includes all kinds of calls" makes it sound like `Call` has some extra things, but that's not the case right?
```suggestion
    NOTE: Consider using `Call` instead, as that excludes calls that are instantiations
    of tuple-like structs and tuple-like variants.
```